ppd/ppd-test.c

```diff
@@ -82,48 +83,48 @@ static int valid_path(const char *keyword, const char *path, int errors,
 static int valid_utf8(const char *s);
 
 
-//'ppdTest()' - Test the correctness of PPD files.
+// 'ppdTest()' - Test the correctness of PPD files.
```

Comment:
Could you create a head comment similar to the ones of the other functions in libppd? We will later run an automatic tools (codedoc) over the library which turns these comments into documentation (see also CUPS and the CUPS Programmer's Manual).